amf-shapes/shared/src/main/scala/amf/shapes/internal/spec/jsonschema/semanticjsonschema/transform/PropertyShapeTransformer.scala

```diff
@@ -34,11 +32,12 @@ case class PropertyShapeTransformer(property: PropertyShape, ctx: ShapeTransform
     propertyMapping
   }
 
-  private def transformArray(array: ArrayShape) = {
+  private def transformArray(array: ArrayShape): Unit = {
```

Comment:
What about array of arrays ?

Reply:
Same as the previous. We should accept it at root level, because we need to extract it to a declaration with the current behavior. I will report another issue to support this, it is not trivial to refactor this.